rclrs_tests/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "rclrs_tests"
+version = "0.1.0"
+authors = ["Chris Reid <christopherjreid@outlook.com>"]
+edition = "2021"
+
+[lib]
+path = "src/lib.rs"
+
+[dependencies]
+anyhow = {version = "1", features = ["backtrace"]}
+test_msgs = {version = "*"}
+diagnostic_msgs = {version = "*"}
+nav_msgs = {version = "*"}
+sensor_msgs = {version = "*"}
+stereo_msgs = {version = "*"}
```

Comment:
These packages seem to be unused (except test_msgs), could you remove them?

Reply:
`cargo build` won't run without this. Some sort of transitive dependency error?

Reply:
I can't reproduce this locally, for me everything builds fine (with `colcon` and `cargo`). Can you show me the exact steps and error?